OrbitGl/App.cpp

```diff
@@ -756,7 +757,8 @@ PresetLoadState OrbitApp::GetPresetLoadState(
 }
 
 ErrorMessageOr<void> OrbitApp::OnSaveCapture(const std::string& file_name) {
-  const auto& key_to_string_map = GCurrentTimeGraph->GetStringManager()->GetKeyToStringMap();
+  const auto& key_to_string_map =
+      GCurrentTimeGraph->GetTrackManager()->GetStringManager()->GetKeyToStringMap();
```

Comment:
The hirarchy "TimeGraph -> TrackManager -> StringManager" feels wrong - can't we use the string manager of the time graph directly?

I'm overall not sure why we're passing the string manager around - there is only a single instance of `StringManager` anyways that is already living in `OrbitApp`, and the only point in our code where this is different is in `CaptureDeserializerLoadFuzzer`, where a new `StringManager` is created and passed to the `TimeGraph`, even though an App exists, not sure why it is done like this?

Reply:
I will try to change it as my next PR and if not, make a bug there.